microsoft-edge/devtools-guide-chromium/autofill/index.md

```diff
@@ -0,0 +1,155 @@
+
+![The Autofill tool](./index-images/autofill-tool.png)
+
+
+<!-- ====================================================================== -->
+## Overview
+
+The **Autofill** tool provides a convenient way to automatically fill forms on websites with saved addresses.  The **Autofill** tool in DevTools lets you inspect the mapping between your form fields, predicted autofill values, and saved data.
+
+The **Autofill** tool lets you inspect only the address info.
+
+See also:
+* [Autofill](https://web.dev/learn/forms/autofill) at web.dev.
+
+
+<!-- ====================================================================== -->
+## Save address info to Edge
+<!-- https://developer.chrome.com/docs/devtools/autofill#save-to-chrome -->
+
+By default, Microsoft Edge prompts you to save address info you entered into a web form when you submit it.
+
+<!-- todo: find in Edge Settings -->
+
+<!-- ![A prompt to save address info](todo: autofill-prompt.png) -->
+
+<!-- If there's no such prompt, in Edge, select Customize and control > Password and Autofill > Addresses and more, and turn on Save and fill addresses. You can also add new addresses here. -->
+
+<!-- ![The 'Addresses and more' menu option](todo: addresses-menu-option.png) -->
+
+If you don't have address data saved and don't want to save it, you can use test address data provided by the **Autofill** tool.
+
+
+<!-- ====================================================================== -->
+## Open the **Autofill** tool
+
+By default, the **Autofill** tool automatically opens if DevTools is open and when you autofill a form on a website.  To turn this off, open the panel manually and clear the **Automatically open this panel** checkbox.
+
+
+To manually open the **Autofill** tool:
+
+1. Open DevTools.
+
+1. In the **Activity Bar** at top, select **More tools** > **Autofill**.
+
+
+To open the **Autofill** tool from the **Command Menu**:
+
+1. Open the **Command Menu** by pressing:
+
+   * macOS: **Command+Shift+P**
+   * Windows, Linux: **Control+Shift+P**
+
+   <!-- ![Command Menu with "autofill" entered](todo: command-menu-autofill.png) -->
+
+1. Start typing **autofill**, select **Show Autofill**, and the press **Enter**.
+
+   By default, the **Autofill** tool opens in **Quick View** at the bottom of DevTools.  You can move it to the **Address Bar** at top.
+
+
+<!-- ====================================================================== -->
+## Inspect autofill data
+<!-- https://developer.chrome.com/docs/devtools/autofill#inspect -->
+
+To inspect autofill data:
+
+1. Make sure [Autofill is turned on and you have address info saved](#save-address-info-to-edge) in Microsoft Edge.  Alternatively, you can use test address data as described in the next section.
+
+1. Open DevTools; for example, open [Demo page with accessibility issues](https://microsoftedge.github.io/Demos/devtools-a11y-testing/) in a new window or tab, right-click the page, and then select **Inspect**.
```

Comment:
This is not a great demo page for autofill, because it doesn't have a form which would trigger the browser's autofill feature.
I suggest considering this out of scope for this PR for now. We could create a new demo page and then rewrite this article to match with it, but we should do this in a separate PR. For now, you can probably just comment out everything after the **Open the Autofill tool** section


moved the above PB comment from PR 3378 to here